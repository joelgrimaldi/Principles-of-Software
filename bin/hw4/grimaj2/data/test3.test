r
NA
NA2
r
NA
NA
r
108
109
r
108
108
r
7
7
r
Troy Building
Troy Building
r
7
Troy Building
q