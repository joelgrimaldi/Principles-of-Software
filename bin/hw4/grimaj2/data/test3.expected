First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [NA]
Unknown building: [NA2]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [NA]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [108]
Unknown building: [109]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [108]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Troy Building to Troy Building:
Total distance: 0.000 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Troy Building to Troy Building:
Total distance: 0.000 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Troy Building to Troy Building:
Total distance: 0.000 pixel units.